tests/test_history.py

```diff
@@ -1,96 +1,103 @@
 """Tests the xonsh history."""
+# pylint: disable=protected-access
 from __future__ import unicode_literals, print_function
 import io
 import os
 import sys
 
 import nose
-from nose.tools import assert_equal, assert_true
+from nose import tools as nose_tools
```

Comment:
Yeah, I guess I feel pretty strongly that the "not importing symbols" rule breaks convention with basically every other big and small code base I am used to working on.  Also, it would affect pretty much every python file here.  I am ecstatic to have the linter pass, but I think the style changes would be too different for many of us - myself included!